File: 0048-rotate-image/0048-rotate-image.cpp
class Solution {
public:
    void rotate(vector<vector<int>>& matrix) {
        vector<vector<int>> newMatrix;
        for(int i=0;i<matrix.size();i++){
            vector<int> row;
            for(int j=matrix.size()-1;j>=0;j--){
                row.push_back(matrix[j][i]);
            }
            newMatrix.push_back(row);
        }
        matrix=newMatrix;
    }
};

File: 0048-rotate-image/0048-rotate-image.py
class Solution:
    def rotate(self, matrix: List[List[int]]) -> None:
        """
        Do not return anything, modify matrix in-place instead.
        """
        newMat = []
        for i in range(len(matrix)):
            row = []
            for j in range(len(matrix)-1, -1, -1):
                print(matrix[j][i])
                row.append(matrix[j][i])
            newMat.append(row)
        matrix[:] = newMat